vuepress/docs/next/docs/reference/glossary.md

```diff
@@ -4,43 +4,47 @@
-| Entando Kubernetes integration service (k8s-service) | Provides an abstraction layer between Entando microservices and the APIs exposed by a Kubernetes cluster |
+| Entando Digital Asset | An image, document, or other media file that is uniquely identifiable and stored digitally in a format supported by the Entando Platform |
+| Entando Hub |  A repository (local, remote, public, or private) containing components built with the Entando Platform; A single Entando App Builder can connect to 1 or more Entando Hubs |
+| Entando Identity Management | Entando's Keycloak- and token-based user management and authentication system |
+| Entando Kubernetes integration service (k8s-service) | A function of the Entando Cluster infrastructure custom resource that provides an abstraction layer between Entando microservices and the APIs exposed by Kubernetes |
+| Entando Local Hub | The local component repository in an Entando App Builder used to store and retrieve shareable components for reuse within an Entando Application |
 | Entando Operator | Provides installation and application lifecycle automation for Entando Applications, microservices and required infrastructure services, e.g. databases and Keycloak |
-| Entando Web Content Management System (WCMS) | Entando's web content management system |
-| Local Hub | The local component repository in an Entando App Builder |
+| Entando Platform | The leading open source application composition platform for development with a Kubernetes architecture |
 | Entando Platform Capability (EPC) | A packaged capability that adds functionality to the platform and/or additional UX controls to the App Builder |
+| Entando Plugin | A microservice that exposes APIs reusable by one or more Entando Applications -- TODO: stand on its own or moved to MS def?? |
 | Entando UX Fragment | An HTML block containing Freemarker tags that allow content to be rendered dynamically according to context |
+| Entando Web Content Management System (WCMS) | Entando's lightweight content and digital asset management system that supports headless operations |
 | Entando Widget | A UI element that can be dragged and dropped onto a page, e.g. a snippet of HTML code added from the browser or a micro frontend, which is a specialized widget |
 
+
 ## General Technical Terms
 
 | Term |  Description
 |:--|:--
-| application composition platform (ACP) | Any development platform that supports a catalog of packaged, composable components that can be assembled into applications, where changes to individual components proliferate to the assembly |
+| application composition platform (ACP) | Any development platform that supports the cataloging and management of composable and packaged components, where new components can be added through custom development or imported from existing assets; it governs the life cycles of both the components and the applications built from their modular assembly and deployment |
```

Comment:
Any development platform that manages and catalogs composable packaged components ...

added through custom development or imported from existing assets >> added through custom development or imported << to simplify & assets have a specific meaning

it governs the life cycles of both the components and the applications built from their modular assembly and deployment >> it governs the life cycles of  components and the applications built with them. 

think it's ok to simplify here , it doesn't need to capture all the complexities.

Reply:
assets: from gartner. "imported" on its own is vague and ambiguous - needs to convey some sense of the source and constraints; "from an existing asset" -> a component can be derived as well as copied over in full (subtle info is valuable here)

the whole point of an acp is to enable modularity, both in the initial build and wrt updates; omitting this information takes away value. i agreed with your pov in our meeting that the more detail the better here, to make the definition clear and comprehensive. simplifying too much means someone has to look elsewhere to truly define and understand the term

Reply:
No strong opinion here. What we have is okay and I think it was approved by the team earlier?